AlexWang-16/alexwang-16.github.io
# Fixes #15: responsive navbar not closing when user clicks away from menu

Fixes #15 

# Purpose of update
Fix issue where when user uses mouse to click away from menu, the responsive navbar does not close automatically.

# Change log
* Fixed menu not closing when user clicks outside of menu bar with mouse

# Test Plan
1. On a device with mouse, change browser window to 320px width
2. Navigate to alexwang.ca 
3. Click on hamburger menu icon
4. Click on space outside of hamburger menu icon

Expected result: The user should see that the navbar collapses.

# Additional notes
